crates/bdk/src/wallet/signer.rs

```diff
@@ -912,7 +973,7 @@ impl ComputeSighash for Segwitv0 {
                 } else if psbt_input
                     .redeem_script
                     .as_ref()
-                    .map(Script::is_v0_p2wpkh)
+                    .map(|s| s.as_script().is_v0_p2wpkh())
```

Comment:
No need to call `as_script` explicitly, `ScriptBuf` derefs to `Script` so all the methods on `Script` are available on `ScriptBuf`.